koji/flatten-el8/test.sh

```diff
@@ -1,4 +1,3 @@
-podman run --privileged -v /home/koji-sync/podman:/root/koji-sync/podman centos:8 /bin/sh /root/koji-sync/podman/script.sh
-
-
-rsync -r --delete -e ssh /home/koji-sync/podman/koji/staged/ root@koji.katello.org:/mnt/koji/releases/split/yum/koji-modules/koji/staged/ -v
+podman run --privileged -v ./:/root/koji-sync/podman centos:8 /bin/sh /root/koji-sync/podman/rhel8-split.sh 
```

Comment:
Should this file have a shebang as the first line?

Reply:
Technically, probably.

Reply:
Shebang added.